packages/oui-navbar/README.md

```diff
@@ -37,6 +38,7 @@ Note: All children menus have `.oui-navbar-menu_fixed`. The component is intende
 | active-link   | string    | @?        | true                |                      |           | current active-link of the navbar                                             |
 | main-links    | array     | <?        | true                | _see example below_  |           | array of objects for the items on the left and the toggler (for responsive)   |
 | aside-links   | array     | <?        | true                | _see example below_  |           | array of objects for the items on the right                                   |
+| toggler-links | array     | <?        | true                | _see example below_  |           | array of objects for the responsive menu                                      |
```

Comment:
No doc for toggle-links below?

Reply:
It's already documented with the section `*-links`. There is no particularity for the object structure for this attribute (like for main-links or aside-links). It use the same base.